mmv1/products/storage/terraform.yaml

```diff
@@ -23,6 +23,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           name: "my-bucket"
         primary_resource_name: "fmt.Sprintf(\"tf-test-my-bucket%s\", context[\"random_suffix\"])"
+      - !ruby/object:Provider::Terraform::Examples
```

Comment:
Because this resource is handwritten, we also don't get autogenerated docs and tests using these examples. See the provider diff that is generated here, which does not include new tests: https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-6870-old..auto-pr-6870

I think you'll need to remove the addition here in `terraform.yaml`, remove `storage_bucket_autoclass.tf.erb`, and instead modify `storage_bucket.html.markdown` and `resource_storage_bucket_test.go.erb` by hand.